working/augmentation-libraries/feature-specification.md

```diff
@@ -619,23 +634,24 @@ It is a **compile-time error** if:
 
 *   The original and augmenting declarations do not have the same declared
     types (return type for getters, parameter type for setters, declared type
-    for variables).
+    for variables). This only applies where types are not omitted in the
+    augmenting declaration.
 
-*   An augmenting declaration uses `augmented` when the original declaration has
-    no concrete implementation. Note that all external declarations are assumed
-    to have an implementation provided by another external source, and
+*   An augmenting declaration uses `augmented` when the augmented declaration
+    has no concrete implementation. Note that all external declarations are
+    assumed to have an implementation provided by another external source, and
     otherwise they will throw a runtime error when called. An `abstract`
     variable introduces no implementation.
 
 *   An augmenting variable’s initializer expression uses `augmented` and the
     augmented variable is not a variable with an initializer.
 
-*   A non-writable variable declaration is augmented with a setter. (
-    Instead, the author can declare a *non-augmenting* setter that goes
-    alongside the implicit getter defined by the final variable.)
-    _Non-writable variable declarations are any that does not introduce a
-    setter, including non-`late`  `final` variables, `late final` variables
-    with an initializer, and `const` variables._
+*   A non-writable variable declaration is augmented with a setter. (Instead,
+    the author can declare a *non-augmenting* setter that goes alongside the
+    implicit getter defined by the final variable.) _Non-writable variable
+    declarations are any that does not introduce a setter, including
```

Comment:
Either "does" -> "do" or: "_A non-writable variable is any that does not...".

I'd lean towards the latter. (I've generally found it easier to read specs when they use singular and "any" to refer to any of a range of possible things.)